src/NHibernate/Intercept/AbstractFieldInterceptor.cs

```diff
@@ -76,6 +80,8 @@ public System.Type MappedClass
 
 		#endregion
 
+		// Since v5.3
+		[Obsolete("This method has no more usages and will be removed in a future version")]
 		public ISet<string> UninitializedFields
```

Comment:
It is more a property than a method, and it has a replacement: `GetUninitializedLazyProperties`.
So it would likely be better to write `Please use GetUninitializedLazyProperties extension method instead`.

Reply:
Corrected.